apps/deploy-web/src/queries/useBalancesQuery.ts

```diff
@@ -66,7 +66,8 @@ export function useBalances(address?: string, options?: Omit<UseQueryOptions<Bal
   return useQuery({
     queryKey: QueryKeys.getBalancesKey(address) as QueryKey,
     queryFn: () => getBalances(settings.apiEndpoint, address),
-    enabled: !!address,
+    enabled: !!address && !!settings.apiEndpoint,
```

Comment:
Ah that was why the balance was not showing on first render?

Reply:
yes, balance was not refetched when `apiEndpoint` was set by Settings provider

Reply:
https://github.com/akash-network/console/issues/1235